docs/releasenotes.md

```diff
@@ -0,0 +1,83 @@
+# Releasenotes Haal-Centraal-BRP-historie-bevragen
+
+
+## Versie 1.0.0
+
+Alhoewel er nog geen eerdere officiële versie van de Haal-Centraal-BRP-historie-bevragen API als release is uitgebracht zijn de specificaties van deze endpoints als onderdeel van de specificatie van Haal-Centraal-BRP-bevragen het afgelopen jaar beschikbaar geweest via de repository. In de [v0.9.0 release van BRP-bevragen](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v0.9.0) zijn de historie-endpoints nog opgenomen. In de afgelopen maanden hebben we de historie-endpoints in een aparte API ondergebracht. Daarbij zijn nieuwe inzichten die het afgelopen jaar zijn opgedaan opgenomen. Recentelijk hebben we de specificaties ook aangepast op het hergebruik van componenten uit BRP-Bevragen release 1.2.0.
+Dit is een globaal (en niet limitatief) overzicht van wijzigingen die daarvoor zijn doorgevoerd.
+
+### Openapi.yaml :
+
+- server-url is aangepast (omdat het een aparte API is geworden.) --> https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen _**(breaking)**_
+- OperationId's zijn aangepast ivm code-generatie issues en nu in UpperCamelCase. _**(breaking)**_
+- Alle descriptions zijn omgezet naar markdown.
+- Bij properties zijn de maxLength, minLength, pattern en (waar overbodig) de title weggehaald.
+- Descriptions zijn aangepast om beter bij de gebruikersbeleving aan te sluiten, overbodige descriptions zijn verwijderd.
+- Enkele parameternamen zijn aangepast vanwege consistentie met andere Haal-Centraal API's en/of de Design Decision om parameternamen in camelCase op te nemen  
+  - datumvan --> datumVan _**(breaking)**_
```

Comment:
deze wijzigingen zijn nog niet doorgevoerd in de verblijfplaatshistorie.feature. Zie jij kans daarin ook de parameternamen te wijzigen?

Reply:
idem voor partnerhistorie.feature en verblijfstitelhistorie.feature

Reply:
Pak ik op